arrocke/gloss-translation
# feat: add lemma resource data structure

## What has changed



Added `LemmaResource`

## Connected Issues



Closes #162 

## QA Steps



- Run `npm i`
    - There should be no errors
- Run `nx prisma db migrate dev`
    - There should be no errors

## Post-Deployment
